yq-yaml-processor/README.md

```diff
@@ -0,0 +1,49 @@
+# yq-yaml-processor
+
+This composite Github Action (GHA) is aimed to process YAML content by leveraging the capabilities of the [YQ](https://github.com/mikefarah/yq) processor.
+
+## Usage
+
+This composite GHA can be used in any repository.
+
+### Inputs
+| Input name | Description                                                                                                       |      Required      |   Default   |
+| :--------: | :---------------------------------------------------------------------------------------------------------------- | :----------------: | :---------: |
+|  patches   | A list of YAML contents (env, file, inline) with associated YQ expressions to apply (piped in the defined order). | :heavy_check_mark: |             |
+|  options   | Command options passed to the YQ CLI                                                                              |                    | `--inplace` |
+
+### Outputs
+| Output name | Description                                                                              |
+| :---------: | :--------------------------------------------------------------------------------------- |
+|   results   | A JSON array with the results of each YQ command executed for each defined YAML content. |
+
+
+### Workflow Example
+```yaml
+---
+name: example
+on:
+  push:
+jobs:
+  yaml-processing:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: camunda/infra-global-github-actions/yq-yaml-processor@main
+      with:
+        patches: |
```

Comment:
At the beginning I thought `inline` was enough but indentation where not preserved if I needed to get the YAML content from the output of another action (for example) and `env` was the solution.
I will refactor a bit to avoid code duplication.